zypp/PoolQuery.cc

```diff
@@ -998,6 +1004,9 @@ namespace zypp
   PoolQuery::repos() const
   { return _pimpl->_repos; }
 
+  const std::string &
+  PoolQuery::getComment() const
+  { return _pimpl->_comment; }
```

Comment:
The other getter functions don't use get prefix, please change it to use the same style.